conf/messages.en

```diff
@@ -1298,34 +1298,34 @@ field.declaration.items.size = Total number of items
 field.declaration.totalNumberOfItems.totalAmountInvoiced = Total amount of all items
 field.declaration.totalPackageQuantity = Total package quantity
 field.declaration.locations.officeOfExit.circumstancesCode = Express consignment
-field.declaration.parties.representativeDetails.details.eori = Representative's EORI
+field.declaration.parties.representativeDetails.details.eori = Representative’s EORI
 field.declaration.parties.representativeDetails.statusCode = Representative type
-field.declaration.parties.declarationHolders.$.eori = <ol><li>Declaration holder {0}</li><li>Authorised party's EORI</li></ol>
-field.declaration.parties.declarationHolders.$.authorisationTypeCode = <ol><li>Declaration holder {0}</li><li>Authorised party's type</li></ol>
+field.declaration.parties.declarationHolders.$.eori = Authorised party’s EORI of Declaration holder {0}
+field.declaration.parties.declarationHolders.$.authorisationTypeCode = Authorised party’s type of Declaration holder {0}
 field.declaration.borderTransport.meansOfTransportCrossingTheBorderIDNumber = Active means of transport reference
 field.declaration.borderTransport.meansOfTransportCrossingTheBorderType = Active means of transport type
 field.declaration.borderTransport.meansOfTransportCrossingTheBorderNationality = Active means of transport nationality
 field.declaration.borderTransport.modeCode = Mode of transport leaving the border
-field.declaration.parties.carrierDetails.details.eori = Carrier's EORI
-field.declaration.parties.carrierDetails.details.address.fullName = Carrier's name
-field.declaration.parties.carrierDetails.details.address.townOrCity = Carrier's town or city
-field.declaration.parties.carrierDetails.details.address.country = Carrier's country
-field.declaration.parties.carrierDetails.details.address.addressLine = Carrier's address line 1
-field.declaration.parties.carrierDetails.details.address.postCode = Carrier's postcode
+field.declaration.parties.carrierDetails.details.eori = Carrier’s EORI
```

Comment:
Ahhhhh!!!! I never thought I would see this one here. A GDS Standard! :+1:

Reply:
Can't find it on my keyboard...